### DISCUSSION TITLE
Can I use just low 4 bits for an 8-bit 8080 MCU?

### DISCUSSION THREAD

**Discussion by BrandonStudio** [Bodmer/TFT_eSPI#2069] [Q&A - General] [answered]
I'm using ESP32 but short of pins, can I just use 4 or 6 data pins of 8 total (i.e. TFT_D0~TFT_D5)?

Besides, is there any control pin that does not need to connect to ESP32 (RST, DC, WR, RD)?

**Comment by Bodmer** [comment, marked as answer]
No, 8 bit data plus WR and DC is minimum. Then CS tied low, RD tied high, RST eithet tied high or to power on reset signal.

**Reply by BrandonStudio** [reply]
Then, am I just undef CS and RD?

**Reply by Bodmer** [reply]
Yes, do not define them and then pins will not be allocated.

**Comment by BrandonStudio** [comment]
Thanks
